.github/workflows/pylint.yml

```diff
@@ -0,0 +1,31 @@
+name: Code Format Check
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8
+          if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
+
+      - name: Lint
+        run: |
+          flake8 --max-line-length=120 --ignore=F401,W503 --count --show-source --statistics orsopy
+          flake8 --max-line-length=120 --ignore=F401,W503 --count --show-source --statistics tests
```

Comment:
I think that if we're suggesting (in CONTRIBUTING.rst) that people call `black` before code is committed, then we should also add the same black test here.

Reply:
@andyfaff I don't think this would be useful. The action would change the code that is tested by flake8 while not changing it in the repository. An auto-commit in the action, on the other hand, could lead to confusion.